### DISCUSSION TITLE
allow wildcard to use lora <lora:MyAwesomeLora-SL:0.90>

### DISCUSSION THREAD

**Discussion by orangpelupa** [runew0lf/RuinedFooocus#130] [Ideas]
feature #17 "Adds the ability to add loras to just prompt ala auto111 style eg <lora:MyAwesomeLora-SL:0.90>" only for the prompt box, would be great to also be supported inside wildcard text file.

**Comment by yownas** [comment]
Having wildcards that pick a random lora from a file? That is an interesting idea. I can't test it at the moment, but I think you are able to create a wildcards file like that already.

Unless things are parsed/added  in the wrong order, but that should be an easy fix.

**Comment by orangpelupa** [comment]
when added to wildcard file, the lora is not imported into the prompt. it simply went "poof". at least according to the prompt shown in the "info" tab
